open.md

```diff
@@ -4,4 +4,111 @@ title: TOKI OSN Open Contest 2019 (English)
+- There will be no additional time given if a contestant starts the contest in less than 5 hours before the contest ends.
+- The open contest doesn't support clarifications, but all important announcements from the original contest will be broadcast as well.
+
+<br>
+
+### Problems Types
+
+- There are 3 possible problem types. There may be some problem type that is not given.
+
+- **Batch** problems:
+  * The most common problem type.
+  * Given input, contestants write a program that reads input from **stdin** and writes output to **stdout**.
+  * The program must run within the given time and memory limits.
+  * Example: [Pekanbaru Defense, OSN 2017](https://training.ia-toki.org/problemsets/91/problems/469/)
+
+- **Interactive** problems:
+  * Contestants write a program that interacts with judge's program: judge's program writes to **stdout** and is given as the **stdin** for contestant's program, then contestant's program writes to **stdout** and is given as the **stdin** for judge's program, and the interaction continues until a given goal is reached.
+  * Contestant's program must output answer within the given time and memory limits.
+  * Example: [Lands and Glaciers, OSN 2017](https://training.ia-toki.org/problemsets/91/problems/471/)
+
+- **Output-only** problems:
+  * Contestants will be given all the input test cases.
+  * Contestants submit **output** for each input test case.
+  * Contestants **do not have to** write a program; the outputs may be solved manually.
+  * Example: [Tourism in Palembang, OSN 2016](https://training.ia-toki.org/problemsets/54/problems/259/).
+
+<br>
+
+### Problem Scoring
+
+* There are two types of scoring: **standard** and **creative**.
+* Creative problems will be stated explicitly in problem description.
+
+* **Standard** scoring:
+  * For batch and interactive problems:
+    * A problem consists of **multiple subtasks** with various points.
+    * A subtask consists of **multiple test cases** that are grouped into some **test groups**.
+    * A submission gets a subtask's points if it solves **all of its test groups**.
+    * To solve a test group, a submission have to solve **all of its test cases**.
+    * A problem may have **open subtasks** where the test cases are given to contestants.
+  * For output-only problems:
+    * A problem consists of **multiple test cases** with various points.
+    * A submission gets the test case's points if it produces the correct output.
+
+* **Creative** scoring:
+  * The scoring formula will vary for each problem and will be explicitly stated in problem description.
+  * Example:
+    * Batch: [Membaca, OSN 2012](https://training.ia-toki.org/problemsets/45/problems/224/).
+    * Interactive: [Cat Rumah, OSN 2014](https://training.ia-toki.org/problemsets/39/problems/202/).
+    * Output-only: [Tourism in Palembang, OSN 2016](https://training.ia-toki.org/problemsets/54/problems/259/).
+
+<br>
+
+### Contestant Scoring
```

Comment:
This does not explain the max score per subtask rule?

Reply:
on line 94,
`* A subtask final score is maximum score of its score among all submissions.`

Reply:
Oh, right